### PR TITLE
add sale_sourced_by_line_sale_addresses

### PR DESCRIPTION
**Pull request by gurneyalex**
this is a glue module between sale_sourced_by_line and sale_addresses (from https://github.com/OCA/stock-logistics-transport/)

Depends on:
- [x] https://github.com/OCA/stock-logistics-transport/pull/6

### CODE REVIEW

**Comment by jgrandguillaume** [comment]
Hi,

Functional behavior is fine. :+1: when build are green.

File: sale_sourced_by_line_sale_transport_multi_address/model/sale_order_line.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+import logging
```
Comment (yvaucher) [inline sale_sourced_by_line_sale_transport_multi_address/model/sale_order_line.py]: ./sale_sourced_by_line_sale_transport_multi_address/model/sale_order_line.py:17:1: F401 'logging' imported but unused

File: .travis.yml [hunk 2]
```diff
@@ -5,14 +5,16 @@ python:
 
 env:
   - VERSION="8.0" LINT_CHECK="1"
-  - VERSION="8.0" ODOO_REPO="odoo/odoo" EXCLUDE="sale_quotation_sourcing,sale_sourced_by_line,sale_quotation_sourcing_stock_route_transit" LINT_CHECK="0"
-  - VERSION="8.0" ODOO_REPO="OCA/OCB" EXCLUDE="sale_quotation_sourcing,sale_sourced_by_line,sale_quotation_sourcing_stock_route_transit" LINT_CHECK="0"
+  - VERSION="8.0" ODOO_REPO="odoo/odoo" EXCLUDE="sale_quotation_sourcing,sale_sourced_by_line,sale_quotation_sourcing_stock_route_transit,sale_sourced_by_line_sale_addresses" LINT_CHECK="0"
```
Comment (yvaucher) [inline .travis.yml]: invalid module names, ignored: sale_sourced_by_line_sale_addresses

-> sale_sourced_by_line_sale_transport_multi_address

File: .travis.yml [hunk 3]
```diff
@@ -5,14 +5,16 @@ python:
...
+  - VERSION="8.0" ODOO_REPO="OCA/OCB" EXCLUDE="sale_quotation_sourcing,sale_sourced_by_line,sale_quotation_sourcing_stock_route_transit,sale_sourced_by_line_sale_addresses" LINT_CHECK="0"
```
Comment (yvaucher) [inline .travis.yml]: invalid module names, ignored: sale_sourced_by_line_sale_addresses

File: sale_sourced_by_line_sale_transport_multi_address/README.rst [hunk 4]
```diff
@@ -0,0 +1,33 @@
+Sale Address Management with multiple warehouse sourcing
```
Comment (yvaucher) [inline sale_sourced_by_line_sale_transport_multi_address/README.rst:4]: Please add License badge

File: sale_sourced_by_line_sale_transport_multi_address/__openerp__.py [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+#
+
+{'name': 'Sale Addresses with multiple Warehouse sources',
+ 'summary': 'Make sale_sourced_by_line and sale_addresses work together',
```
Comment (yvaucher) [inline sale_sourced_by_line_sale_transport_multi_address/__openerp__.py]: s/sale_addresses/sale_multi_address

**Comment by lepistone** [comment]
I addressed @yvaucher's remarks.

**Comment by yvaucher** [comment]
:+1:

**Comment by yvaucher** [comment]
Thanks @lepistone for the good catch on the file name !

**Pull request merged**